### DISCUSSION TITLE
Temas para Nuevo Video

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#581] [Ideas]
De que va a hablar cada uno y que tener en cuenta para el tema elegido

**Comment by rpgrca** [comment]
Podemos hablar de lo mismo de la otra vez o podemos mezclar un poco para que cada uno hable de algo distinto, qué prefieren?

**Reply by NataliaAlvarezIspc** [reply]
Me parece que podríamos hablar de las mejoras que hicimos para este sprint, como que ahora en Perfil se pueden modificar los datos o que ahora devuelve las ventas para el administrador y las compras para el cliente, etc.  Para no repetir con respecto a video anterior, pero como le parezca al resto.

**Reply by BraianTroncoso** [reply]
Me gustan ambas ideas, por ahi puede ser lo mismo pero profundizando en si, ¿qué dicen?

**Comment by NataliaAlvarezIspc** [comment]
Estos son los requerimientos del sprint  4

![image](https://github.com/NataliaAlvarezIspc/proyecto-ispc-ecommerce/assets/106571110/f85677f9-531a-45b0-bc52-83ee27fa229a)
